pymc_marketing/mmm/lift_test.py

```diff
@@ -63,8 +62,19 @@ def lift_test_indices(df_lift_test: pd.DataFrame, model: pm.Model) -> Indices:
     """
 
     columns = df_lift_test.columns.tolist()
+
+    def _ensure_datetime_type_in_array(x: np.ndarray) -> np.ndarray:
```

Comment:
I suggest changing the name to _coerce_datetype_type_in_array and being more explicit with types

```suggestion
    def _coerce_datetime_type_in_array(x: npt.NDArray[Any]) -> npt.NDArray[np.datetime64]:
```

Reply:
Sorry. Missed this
Ended up changing the logic a bit and removed